modules/customize-certificates-add-service-serving-configmap.adoc

```diff
@@ -32,23 +32,24 @@ $ oc annotate configmap <configmap-name> \//<1>
 +
 [NOTE]
 ====
-Explicitly referencing the `service-ca.crt` key in a volumeMount will
-prevent a Pod from starting until the ConfigMap has been injected with
-the CA bundle.
+Explicitly referencing the `service-ca.crt` key in a volumeMount will prevent a Pod from starting until the ConfigMap has been injected with the CA bundle. This behavior can be overridden by setting the `optional` field to `true` for the volume's serving certificate configuration.
 ====
 +
-For instance, to annotate the ConfigMap `foo` the following command would be
-used:
+For example, use the following command to annotate the ConfigMap `foo`:
 +
 ----
 $ oc annotate configmap foo service.beta.openshift.io/inject-cabundle=true
 ----
 
-. View the ConfigMap to ensure the certificate has been generated. This
-appears as a `service-ca.crt` in the YAML output.
+. View the ConfigMap to ensure that the certificate has been generated:
 +
 ----
 $ oc get configmap <configmap-name> -o yaml
+----
++
+It is displayed as a `service-ca.crt` in the YAML output:
```

Comment:
The CA bundle will appear as the value of the `service-ca.crt` key in the YAML output